documentation/contributing/code/index.markdown

```diff
@@ -114,19 +120,33 @@ In the package settings set ``clang-format-3.9`` as your executable and point 'S
 more modern, more readable, and less prone to common bugs.
 
 You can install clang-tidy and other clang related tools with
-`sudo apt-get install clang libclang-dev clang-tidy-3.9 clang-format-3.9`
+`sudo apt-get install clang-tidy clang-tools`
```

Comment:
So is clang-format's version still 3.9, but clang-tidy's is 6.0? That seems a little strange.

Reply:
I see. And there's no reason to keep clang-tidy to the same version.

Reply:
Probably, clang-tidy will behave differently in different versions as well. But here, I think it's to our best: the 6.0 version will find (and fix) more stuff and 3.x should not complain about these.